lib/api/route/streams.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:json_annotation/json_annotation.dart';
+
+import '../core.dart';
+part 'streams.g.dart';
+
+/// https://zulip.com/api/v1/user/me/:stream_id/topics
+Future<GetTopicsResult> getStreamTopics(
```

Comment:
This should link to the corresponding API docs page — that's the page the reader will want to refer to in order to understand the semantics of this function.